src/lib/deso/transaction.ts

```diff
@@ -418,6 +418,7 @@ export class TransactionDAOCoinLimitOrderLimitMapItem extends BinaryRecord {
   value: number = 0;
 }
 
+// TODO: does this need to be updated? Where is it really used?
```

Comment:
Isn't this used to serialize/deserialize the metadata of a txn in the FE? I think it doesn't need to be updated because we're recycling the same struct for lockups right?

Reply:
It's not actually used anywhere. It's referenced, but these references are unused.